waves-integration-it/src/test/scala/com/wavesplatform/dex/grpc/integration/sync/DEXExtensionTestSuite.scala

```diff
@@ -0,0 +1,153 @@
+package com.wavesplatform.dex.grpc.integration.sync
+
+import com.google.protobuf.empty.Empty
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.account.Address
+import com.wavesplatform.common.utils.EitherExt2
+import com.wavesplatform.dex.grpc.integration.dto.{BalanceChangesResponse => VBalanceChangesResponse}
+import com.wavesplatform.dex.grpc.integration.protobuf.BalancesServiceConversions
+import com.wavesplatform.dex.grpc.integration.protobuf.Implicits._
+import com.wavesplatform.dex.grpc.integration.services.BalancesServiceGrpc.BalancesServiceStub
+import com.wavesplatform.dex.grpc.integration.services.CurrentObservableBalancesResponse.ResponseMapEntry
+import com.wavesplatform.dex.grpc.integration.services.{BalanceChangesRequest, BalanceChangesResponse, BalancesServiceGrpc}
+import com.wavesplatform.it.Docker
+import com.wavesplatform.it.api.SyncHttpApi._
+import com.wavesplatform.it.sync.{issueFee, minFee, someAssetAmount}
+import com.wavesplatform.it.transactions.BaseTransactionSuite
+import com.wavesplatform.it.util._
+import com.wavesplatform.transaction.Asset
+import com.wavesplatform.transaction.Asset.Waves
+import io.grpc.stub.StreamObserver
+import io.grpc.{ManagedChannel, ManagedChannelBuilder}
+import mouse.any._
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+class DEXExtensionTestSuite extends BaseTransactionSuite {
```

Comment:
`BaseTransactionSuite` is a base class for NODE's integration tests, it doesn't know about Kafka/Local queue. All tests use Kafka queue by default and everyday at night we run all integration tests on a local queue.

So, this test should extend `MatcherSuiteBase`.

Reply:
`waves-integration-it` depends on `waves-integration`
`waves-integration` depends on NODE, not DEX

*roll_safe*